docs/source/tutorial/introduction.md

```diff
@@ -1,74 +1,80 @@
 
-## Set up your development environment
+- [Node.js](https://nodejs.org/) v8.x or later
+- [npm](https://www.npmjs.com/) v6.x or later
+- [git](https://git-scm.com/) v2.14.1 or later
 
-Now the fun begins! First, you'll need to install our developer tools:
+Although it isn't required, we also recommend using [VS Code](https://code.visualstudio.com/)
+as your editor so you can use Apollo's helpful VS Code extension.
 
-- [Apollo Graph Manager (required)](https://engine.apollographql.com) : Our cloud service where you'll register and manage your graph API.
-- [Apollo DevTools for Chrome (suggested)](https://chrome.google.com/webstore/detail/apollo-client-developer-t/jdkknkkbebbapilgoeccciglkfbmbnfm) : Our Chrome extension giving you full visibility into your client.
-- [Apollo VSCode (suggested)](https://marketplace.visualstudio.com/items?itemName=apollographql.vscode-apollo): Our editor integration that offers intelligent autocomplete, metrics, and more.
+## Clone the example app
 
-Next, in your terminal, clone [this repository](https://github.com/apollographql/fullstack-tutorial):
+Now the fun begins! From your preferred development directory, clone [this repository](https://github.com/apollographql/fullstack-tutorial):
 
 ```bash
 git clone https://github.com/apollographql/fullstack-tutorial/
 ```
 
-There are two folders: one for the starting point (`start`) and one for the final version (`final`). Within each directory are two folders: one for the server and one for the client. We will be working in the server folder first. If you're comfortable with building a graph API already and you want to skip to the client portion, navigate to the [last half of the tutorial](/tutorial/client/).
+The repository contains two top-level directories: `start` and `final`. During the
+tutorial you'll edit the files in `start`, and at the end they'll match the
+completed app in `final`.
 
-<!--
-TODO: Add in this section after Apollo VSCode works for server development
-### Configure Apollo VSCode -->
+Each top-level directory contains two directories of its own: `server` and `client`. We'll be working in the `server` directory first. If you're already comfortable
+building a graph API and you want to skip to the `client` portion, navigate to the [second half of the tutorial](/tutorial/client/).
 
-### Where can I get help?
+## Need help?
 
-We know that learning a new technology can sometimes be overwhelming, and it's totally normal to get stuck! If that happens, we recommend joining the [Apollo Spectrum](https://spectrum.chat/apollo) community and posting in the relevant channel (either #apollo-server or #apollo-client) for some quick answers.
+Learning a new technology can be overwhelming sometimes, and it's easy to get stuck! If that happens, we recommend joining the [Apollo Spectrum community](https://spectrum.chat/apollo) and posting in the relevant channel (either `#apollo-server` or `#apollo-client`) for assistance from friendly fellow developers.
```

Comment:
```suggestion
Learning a new technology can be overwhelming sometimes, and it's common to get stuck! If that happens, we recommend joining the [Apollo Spectrum community](https://spectrum.chat/apollo) and posting in the relevant channel (either `#apollo-server` or `#apollo-client`) for assistance from friendly fellow developers.
```

We should consider eliminating the words "easy/simple" from the docs.

Reply:
Haha certainly agreed; my brain gave this one a pass because it's describing how easy it is to struggle as opposed to succeed, but we might as well eliminate in all cases.